services/ymax-planner/src/main.ts

```diff
@@ -218,6 +233,37 @@ export const main = async (
     makeNonce: () => new Date(now()).toISOString(),
   });
 
+  const fetchAccountInfo: FetchAccountInfo = async address => {
+    return cosmosRest.getAccountSequence('agoric', address);
+  };
+
+  const sequenceManager = await makeSequenceManager(
+    {
+      log: (...args) => console.log('[SequenceManager]:', ...args),
+    },
+    {
+      address: signingSmartWalletKit.address,
+      fetchAccountInfo,
+    },
+  );
+
+  const smartWalletWithSequence = makeSmartWalletWithSequence(
+    {
+      signingSmartWalletKit,
+      sequenceManager,
+      log: (...args) => console.log('[SmartWalletWithSequence]:', ...args),
```

Comment:
```suggestion
      log: (...args) => console.log('[SigningSmartWallet]', ...args),
```
But also, this seems likely to be excessively noisy. I wonder if most of the logging from `makeSmartWalletWithSequence` should be hidden behind a `verbose` option.